### PR TITLE
fix(docs): Update `@astrojs/sitemap` readme to clarify build output location

### PR DESCRIPTION
**Pull request by evadecker**
## Changes

- This slightly modifies the language in the docs to clarify that `/dist` is the default output, but generated files may live elsewhere (a confusion that cost me a silly amount of time trying to debug).
- I considered adding a note that the Vercel adapter will output to `.vercel/output` and that other adapters may output code to a place other than `/dist`, but this felt too in-the-weeds. If this is helpful to call out, happy to make edits!
- Resolves https://github.com/withastro/docs/issues/4919 (~~Not sure if Github will auto-close this PR since it's in a separate repo, but the sitemap docs get auto-generated from this README~~)
- Also pairs with @silent1mezzo's improvements to the CLI text: https://github.com/withastro/astro/pull/8824

## Testing

Docs-only change.

### CODE REVIEW

**Comment by ematipico** [comment]
> * (Not sure if Github will auto-close this PR since it's in a separate repo, but the sitemap docs get auto-generated from this README)

It seems it will :)

**Comment by ematipico** [comment]
@withastro/maintainers-docs this is your turf, please review :)

**Review by newtoallofthis123 — COMMENTED** [review]
lgtm 😄

File: packages/integrations/sitemap/README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ export default defineConfig({
 
 Note that unlike other configuration options, `site` is set in the root `defineConfig` object, rather than inside the `sitemap()` call.
 
-Now, [build your site for production](https://docs.astro.build/en/reference/cli-reference/#astro-build) via the `astro build` command. You should find your sitemap under `dist/` for both `sitemap-index.xml` and `sitemap-0.xml`!
+Now, [build your site for production](https://docs.astro.build/en/reference/cli-reference/#astro-build) via the `astro build` command. You should find both `sitemap-index.xml` and `sitemap-0.xml` in your output directory—by default, `/dist`.
```
Comment (sarah11918) [inline packages/integrations/sitemap/README.md:81]:
```suggestion
Now, [build your site for production](https://docs.astro.build/en/reference/cli-reference/#astro-build) via the `astro build` command. You will find both `sitemap-index.xml` and `sitemap-0.xml` in the `dist/` folder (or your custom [output directory](https://docs.astro.build/en/reference/configuration-reference/#outdir) if set).
```
What do you think about something like this as being a little smoother, and linking here to the output directory config option? 

I also considered just: 

> in your [output directory](https://docs.astro.build/en/reference/configuration-reference/#outdir).

but I think maybe enough people will just use the default `dist/` folder that mentioning it explicitly is still helpful?

**Review by sarah11918 — COMMENTED** [review]
Thank you so much for this helpful restructuring, @evadecker!  🥳 

It's a great tweak to the existing sentence to more properly identify the sitemap files. I'd like to work on the ending though, since we don't typically use em-dashes in docs. That probably means either sticking another clause on the end, or using parenthesis to include the extra information of the default output directory. 

I left a comment with a suggestion below. See what you think, or if that gives you any better ideas.  🙌

**Comment by evadecker** [comment]
> Thank you so much for this helpful restructuring, @evadecker! 🥳
> 
> It's a great tweak to the existing sentence to more properly identify the sitemap files. I'd like to work on the ending though, since we don't typically use em-dashes in docs. That probably means either sticking another clause on the end, or using parenthesis to include the extra information of the default output directory.
> 
> I left a comment with a suggestion below. See what you think, or if that gives you any better ideas. 🙌

I respect killing the em dash—I overuse it. :)

Great suggestion @sarah11918, applied!

**Review by ematipico — APPROVED** [review]

**Pull request merged**